.lycheeignore

```diff
@@ -4,8 +4,8 @@
 
 my-company.com
 https://megalinter.io/
-https://megalinter.io/flavors/
-https://megalinter.io/configuration/
```

Comment:
These seem like exclusions to not error out. Are they the ones you want to fix?

Reply:
Yes, that was intentional. If the wrong URLs no longer appear anywhere, they don't have to be in the .lycheeignore.